controller/src/comms.cpp

```diff
@@ -20,15 +20,15 @@ limitations under the License.
  *    PRIVATE FUNCTION PROTOTYPES
  ****************************************************************************************/
 
-static bool packet_receive(char *packet, uint8_t *packet_len);
-static bool packet_checksumValidation(char *packet, uint8_t len);
-static bool packet_cmdValidatation(char *packet);
-static bool packet_modeValidation(char *packet);
-static enum processPacket process_packet(char *packet, uint8_t len);
-static void comms_sendModeERR(char *packet);
-static void comms_sendChecksumERR(char *packet);
-static void comms_sendCommandERR(char *packet);
-static void send_alarm();
+bool packet_receive(char *packet, uint8_t *packet_len);
```

Comment:
Shouldn't these still be static?  Instead of removing `static` here, do we want to be adding it to the functions themselves?

Reply:
...actually, I see that we removed `static` from the functions too.  Why?